GSoC-Participants.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: GSoC participants
+---
+
+This document collects the list of contributors who've contributed
+to Git via GSoC.
+
+<!-- [ [project]() ] [ [final report]() ] [ [blog]() ] -->
+
+### 2023
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2023/organizations/git)
+
+### 2022
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2022/organizations/git)
+
+### 2021
+
+*TODO*
+
+#### References
+
+- [GSoC archive](https://summerofcode.withgoogle.com/archive/2021/organizations/6398200235163648)
```

Comment:
From _posts/2021-05-27-edition-75.markdown:

  - ZheNing Hu will work on the
    [Use ref-filter formats in `git cat-file`](https://summerofcode.withgoogle.com/projects/#6499923274498048)
    project. He will be co-mentored by Hariom Verma and Christian
    Couder. His first blog post about it [is up](https://adlternative.github.io/GSOC-Git-Blog-1/).

  - Atharva Raykar will work on the
    [Finish converting submodule to builtin](https://summerofcode.withgoogle.com/projects/#4754589292691456)
    project. He will be co-mentored by Shourya Shukla and Christian
    Couder. His first blog posts about it are [also up](https://atharvaraykar.me/gitnotes/).